### PR TITLE
update azure_rm_aks document

### PR DESCRIPTION
**Pull request by Fred-sun**
##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

azure_rm_aks/azure_rm_aksversion_facts/azure_rm_aks_facts 
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by Fred-sun** [comment]
@yuwzho Please help take a look this PR when you're available! Thanks a lot!

File: lib/ansible/modules/cloud/azure/azure_rm_aks.py [hunk 1]
```diff
@@ -114,7 +114,7 @@
                 description: Network policy used for building Kubernetes network.
             pod_cidr:
                 description:
-                    - A CIDR notation IP range from which to assign pod IPs when kubenet is used.
+                    - A CIDR notation IP range from which to assign pod IPs when C(kubenet) is used.
```
Comment (yuwzho) [inline lib/ansible/modules/cloud/azure/azure_rm_aks.py]: when I(neteork_plugin=kubenet)

**Review by yuwzho — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/cloud/azure/azure_rm_aks.py [hunk 2]
```diff
@@ -47,32 +47,32 @@
...
         suboptions:
             admin_username:
                 description:
-                  - The Admin Username for the Cluster.
+                    - The Admin Username for the Cluster.
                 required: true
             ssh_key:
                 description:
-                    - The Public SSH Key used to access the cluster.
+                    - The Public SSH Key used to access the Cluster.
```
Comment (yuwzho) [inline lib/ansible/modules/cloud/azure/azure_rm_aks.py]: cluster. 

If not a term, no need to capitalize it.

File: lib/ansible/modules/cloud/azure/azure_rm_aks_facts.py [hunk 3]
```diff
@@ -18,7 +18,7 @@
 
 version_added: "2.6"
 
-short_description: Get Azure Kubernetes Service facts.
+short_description: Get Azure Kubernetes Service facts
```
Comment (yuwzho) [inline lib/ansible/modules/cloud/azure/azure_rm_aks_facts.py:21]: Why remove it?

**Review by yuwzho — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/cloud/azure/azure_rm_aks_facts.py (hunk 3 above)
Comment (Fred-sun) [reply lib/ansible/modules/cloud/azure/azure_rm_aks_facts.py:21]: @yuwzho Here is a little hint from Ansible document, short_description should not have a trailing period/full stop([Link](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-within-module-documentation)). Thanks a lot!

File: lib/ansible/modules/cloud/azure/azure_rm_aks.py [hunk 4]
```diff
@@ -69,6 +69,7 @@
                 description:
                     - Number of agents (VMs) to host docker containers.
                     - Allowed values must be in the range of C(1) to C(100) (inclusive).
+                default: 1
```
Comment (yuwzho) [inline lib/ansible/modules/cloud/azure/azure_rm_aks.py]: https://github.com/ansible/ansible/blob/14f5283effaa0219336e74603870bd85e3221385/lib/ansible/modules/cloud/azure/azure_rm_aks.py#L413

There is no default value, and this field is required.

**Review by yuwzho — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/cloud/azure/azure_rm_aks.py (hunk 4 above)
Comment (Fred-sun) [reply lib/ansible/modules/cloud/azure/azure_rm_aks.py]: Thanks yuwzho, I will notice like this define and modify it!

**Comment by Fred-sun** [comment]
@yuwzho The shippable check fail, but the error log show nothing, all test item was cancelled.  Could you can tell the reason. Could you help take a look this? Thank you very much!

**Comment by yuwzho** [comment]
no idea

**Comment by yuwzho** [comment]
but your PR contains merge commit

**Comment by Fred-sun** [comment]
But all check passed before this commit!

**Comment by yuwzho** [comment]
Besides the CI, you need to fix the merge commits anyway.

**Comment by Fred-sun** [comment]
Thanks Yuwzho, I work on this!

**Comment by Fred-sun** [comment]
I will close it, re-open a new instead of this! Thanks a lot!. new PR #56293 . Thanks a lot!

**Pull request closed without merging**